src/rnp/rnpcfg.c

```diff
@@ -116,8 +118,12 @@ rnp_cfg_find(rnp_cfg_t *cfg, const char *key)
     return (i == cfg->count) ? -1 : (int) i;
 }
 
-/* resize keys/vals arrays to the new size. Only expanding is supported */
-static int
+/** @brief resize keys/vals arrays to the new size. Only expanding is supported now.
```

Comment:
It's just a static function now, and used only in one place. So let it be called resize, this will not make impact on anything.